src/Yaapii.Xml/XMLCursor.cs

```diff
@@ -45,95 +45,117 @@ public sealed class XMLCursor : IXML
         private readonly IScalar<string> xml;
         private readonly IScalar<XNode> cache;
         private readonly IScalar<IXmlNamespaceResolver> context;
-        private readonly Func<XNode, string> stringTransform;
-
+        private readonly Func<XNode, string> stringTransform;
+
         /// <summary> 
-        /// XMLCursor from Xambly Directives. 
+        /// A XML Cursor, which can be set to different nodes via XPath queries. Type is <see cref="IXML"/>.
         /// </summary>
-        /// <param name="patch">Xambly patch</param>
         public XMLCursor(IEnumerable<IDirective> patch) : this(
             new Xambler(patch)
         )
-        { }
-
-        /// <summary> XMLCursor from a Xambler. </summary>
-        /// <param name="xambler"> Xambler to make Xml from </param>
+        { }
+
+        /// <summary> 
+        /// A XML Cursor, which can be set to different nodes via XPath queries. Type is <see cref="IXML"/>.
+        /// </summary>
         public XMLCursor(Xambler xambler) : this(
             new TextOf(
                 new ScalarOf<string>(() => xambler.Xml())
             )
         )
-        { }
-
-        /// <summary> XMLCursor from a XNode. </summary>
-        /// <param name="node"> XNode to make XML from </param>
+        { }
+
+        /// <summary> 
+        /// A XML Cursor, which can be set to different nodes via XPath queries. Type is <see cref="IXML"/>.
+        /// </summary>
         public XMLCursor(XNode node) : this(
             new ScalarOf<XNode>(node),
             new ScalarOf<IXmlNamespaceResolver>(new XPathContext())
         )
-        { }
-
-        /// <summary> XMLCursor from a stream. </summary>
-        /// <param name="stream"> stream with xml text </param>
-        public XMLCursor(Stream stream) : this(stream, Encoding.Default)
-        { }
-
-        /// <summary> XMLCursor from a stream. </summary>
-        /// <param name="stream"> stream with xml text </param>
+        { }
+
+        /// <summary> 
+        /// A XML Cursor, which can be set to different nodes via XPath queries. Type is <see cref="IXML"/>.
+        /// </summary>
+        public XMLCursor(Stream stream) : this(
+            stream,
+            Encoding.Default
```

Comment:
i suppose this is no code execution but i am not sure. do you know?

Reply:
'+ more cases below

Reply:
Accessing enumerations is only referring to an address, so it does not execute code. I accesses a constant value already present in memory. The compiler will replace the line with the value behind the enumeration.